frontend/composables/useMSAL.ts

```diff
@@ -19,7 +19,7 @@ import { computed } from 'vue'
 import { type APIKey } from '~/openapi/generated/user'
 
 export const useMSAL = async () => {
-  const isAuthenticated = useState('useMSAL.isAuthenticated', () => false)
+  const { isAuthenticated } = useSession()
```

Comment:
This creates a dependency loop:

- `useSession` depends on `usePACTA`
- `usePACTA` depends on `useMSAL`
- `useMSAL` depends on `useSession`

In this case, you should be able to unravel it just by moving `isAuthenticated` back into `useMSAL` (basically just flipping the dependency arrow), but if that doesn't work for some reason, let's talk about it because having loops like this tends to break suddenly and dramatically in hard to debug ways.

Reply:
Fixed.